### ISSUE TITLE
main.ipynb

### ISSUE THREAD

**Issue by Shahnawazgrewal** [vdyashin/LearnablePINs#20]
Can you please check `main.ipynb` file, there is some  issue with file, can't open

**Comment by v-iashin** [comment]
Hi. Thanks for the interest in my implementation and for the catch. You are right.

I think I localized the problem. Specifically, commit with id `2420e5` somehow messed up with the `.ipynb` file since `d47a433` works ok.

I made a commit which fixes it. I also check out whether it is openable in nbviewer: https://nbviewer.jupyter.org/github/v-iashin/LearnablePINs/tree/e143c6a69f4e234f60ed2a48991a63b7b1e6341c/

Please let me know if you find any other problem.

**Comment by Shahnawazgrewal** [comment]
I am aware that the code is not complete, but what sort of performance you obtained?

**Comment by v-iashin** [comment]
none, it is not complete 🙂

**Comment by v-iashin** [comment]
I mean, it doesn't converge

**Comment by Shahnawazgrewal** [comment]
I am trying to use slightly different approach approach. Once I have the embeddings from `faces` and `audio` , I am using a shallow architecture to bridge the gap between the top using softmax and center loss.

**Comment by v-iashin** [comment]
Ok, can you be more specific? Why have you decided to use a different loss? What exactly your "shallow architecture" inputs ([`emb_a`, `emb_v`]?) and outputs (`emb_av`?).

**Comment by Shahnawazgrewal** [comment]
yes exactly! that is the input and output of my model.
I am doing slightly a different project so I opted to introduce another loss function!

**Comment by v-iashin** [comment]
Thank you for sharing. Good luck with your project. I hope you will find useful ideas out of this implementation.

**Comment by Shahnawazgrewal** [comment]
I will share the implementation with you very soon!
